<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDialog</class>
 <widget class="QFrame" name="MessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MessageDialog {
	background-color : rgba(0,0,0,50%);
}

#messageDialogWidget {
	background-color : white;
	border : 1px solid white;	
	border-radius : 6%;
}</string>
  </property>
  <widget class="QWidget" name="messageDialogWidget" native="true">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>135</y>
     <width>769</width>
     <height>499</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>769</width>
     <height>499</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <widget class="ClickableQWidget" name="close_button_container" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="close_button" native="true">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#close_button {
	background-image: url(:/assets/images/cancel_2.png);
	background-repeat: no-repeat;
	background-position: center;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>70</y>
      <width>771</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="messageDialogContainer"/>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableQWidget</class>
   <extends>QWidget</extends>
   <header location="global">clickableqwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
